### PR TITLE
Net standard docs

### LINKED ISSUE #10556: Netstandard contract list needs to be updated to be correct.
https://github.com/dotnet/corefx/blob/master/Documentation/architecture/net-platform-standard.md needs to be updated to represent what is actually shipped. There are some things in there that are not correct like the PInvoke contract that never shipped and was removed.

### PR DESCRIPTION
**Pull request by ericstj**
Update the NETStandard docs with API versions and TFMs mappings.

Though we aren't specifically using ns1.7 to slot in any API (due to the 2.0 plans) we should update the docs to represent it.

Fixes #10556

/cc @Petermarcu

### CODE REVIEW

File: Documentation/architecture/net-platform-standard.md [hunk 1]
```diff
@@ -50,24 +50,24 @@ Each platform has it own rules for how it anchors certain assemblies. In .NET Fr
...
+|||&rarr;|&rarr;|4.5.1||||||
+|||&rarr;|4.5|||||||
+|Universal Windows Platform|uap|&rarr;|&rarr;|&rarr;|&rarr;|10.0||||
+|Windows|win|&rarr;|&rarr;|8.1||||||
+|||&rarr;|8.0|||||||
+|Windows Phone|wpa|&rarr;|&rarr;|8.1||||||
+|Windows Phone Silverlight|wp|8.1||||||||
+|||8.0||||||||
+|Mono/Xamarin Platforms||&rarr;|&rarr;|&rarr;|&rarr;|&rarr;|&rarr;|&rarr;|*|
+|Mono||&rarr;|&rarr;|*|||||||
```
Comment (svick) [inline Documentation/architecture/net-platform-standard.md]: I don't see the new column in the table as rendered by GitHub. I believe you also need to add a new column on the very first row of the table.

**Comment by weshaggard** [comment]
I think it is worth adding a note about NS1.7 if you plan to put it into the table.

**Comment by karelz** [comment]
@ericstj any update on the PR status?

**Comment by karelz** [comment]
@ericstj ping?

**Comment by ericstj** [comment]
Was waiting to merge because 1.1 was in lock down.  It can be merged once 1.1 is ready to accept.

**Comment by karelz** [comment]
Ah, I missed that detail, sorry - will mark it with 1.1.0 milestone as well.

**Pull request merged**